packages/web/src/components/cells/code.tsx

```diff
@@ -447,7 +447,7 @@ function Header(props: {
                     onClick={formatCell}
                     tabIndex={1}
                   >
-                    <PrettierLogo size={16} />
+                    <PrettierLogo size={14} />
```

Comment:
the prettier icon does take up space that is usually white space in the others which makes it appear bigger even if the total size in the DOM is the same

Reply:
hmm ok I guess I didn't notice, happy to keep this change

Reply:
Yeah, this is eye test over numbers, it doesn't line up (icon sets have padding conventions, so since this isn't lucide it's not consistent with the others)